### PR TITLE
Update .travis.yml

### PR DESCRIPTION
**Pull request by pschambacher**
:octopus::computer:

/cc @grosser 

### Steps to merge
* [ ] :+1: from the team

### Risks
* None

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,16 @@
+sudo: false
+
 language: ruby
+
+bundler_args: ''
```
Comment (grosser) [inline .travis.yml]: kills the caching :)

File: .travis.yml [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
   bundler: true
+
 branches:
   only: master
-sudo: false
+
 rvm:
-  - 2.0.0
-  - 2.1.6
-  - 2.2.2
```
Comment (grosser) [inline .travis.yml]: bump the ruby requirement if we stop testing on 2.0 ?

File: .travis.yml (hunk 1 above)
Comment (pschambacher) [inline .travis.yml]: We have this everywhere. I think that using bundler_args doesn't kill caching, having a custom bundle install does.

File: .travis.yml (hunk 2 above)
Comment (pschambacher) [inline .travis.yml]: Can do.

File: .travis.yml (hunk 1 above)
Comment (grosser) [inline .travis.yml]: hmm it does a `bundle install  --path=${BUNDLE_PATH:-vendor/bundle}` so that might work out just fine ...

**Comment by grosser** [comment]
👍

**Pull request merged**